Sources/TuistGenerator/Graph/Graph.swift

```diff
@@ -309,15 +309,6 @@ class Graph: Graphing {
 
         references.append(contentsOf: otherTargetFrameworks)
 
-        /// Pre-built frameworks
```

Comment:
[Line 299](https://github.com/tuist/tuist/pull/490/files#diff-12f3cd68bf1720345626c27c16f99056R299) takes care of embedding pre-compiled nodes (which include `FrameworkNode`) while checking their linkage (dynamic vs static)